specs/vep-1243-vdk-meta-jobs/README.md

```diff
@@ -34,6 +34,15 @@ service. Meta Jobs provide a more lightweight alternative that simplifies the pr
 Optional section which defines terms and abbreviations used in the rest of the document.
 -->
 
+* **VDK**: https://github.com/vmware/versatile-data-kit/wiki/dictionary#vdk
+* **Control Service**: https://github.com/vmware/versatile-data-kit/wiki/dictionary#control-service
+* **Data Job**: https://github.com/vmware/versatile-data-kit/wiki/dictionary#data-job
+* **Data Job Step**: https://github.com/vmware/versatile-data-kit/wiki/dictionary#data-job-step
+* **Data Job Execution**: https://github.com/vmware/versatile-data-kit/wiki/dictionary#data-job-execution
+* **Apache Airflow (Airflow)**: https://airflow.apache.org/
+* **DAG**: Directed Acyclic Graph
+* **reqs/min**: requests per minute
+
```

Comment:
May link to https://github.com/vmware/versatile-data-kit/wiki/dictionary

Reply:
Do you mean adding a link to the dictionary after all the provided terms and abbreviations or instead of listing them all 1 by 1?

Reply:
No need to duplicate term definitions; you can link to the dictionary

Reply:
Done.